src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -296,6 +298,16 @@ protected void checkAdditionalConstraints() {
         return;
     }
     
+    /**
+     * Note that when overriding this method, this should not use {@code respondingStudentList} 
+     * or {@code respondingInstructorList} of {@code FeedbackSessionAttributes}.
+     * 
+     * @return true if user has responses in the feedback session
+     */
+    protected boolean isUserRespondentOfSession() {
+        return hasValidResponse;
```

Comment:
Right, the method for the single question submission will have to be the default one then